.circleci/config.yml.in

```
@@ -986,7 +986,7 @@ jobs:
             curl -o conda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
             sh conda.sh -b
             source $HOME/miniconda3/bin/activate
-            conda install -yq conda-build cmake
+            conda install -yq conda-build cmake python=3.9
```

For consistency with the other cmake jobs, I think this should be 

```suggestion
            conda install -yq conda-build cmake python=PYTHON_VERSION
```

which would require an extra section above like:

```
    environment:
      PYTHON_VERSION: << parameters.python_version >>
```

---

Good point!